src/main.rs

```diff
@@ -19,6 +23,19 @@ fn parse_size(s: &str) -> usize {
     s.parse().unwrap_or_default()
 }
 
+fn parse_uid(s: &str) -> Option<i16> {
+    assert!(s.starts_with("Uid:"));
+    s[4..]
+        .split_whitespace()
+        .next()
+        .and_then(|uid| uid.parse().ok())
+}
+
+fn get_username(uid: u32) -> String {
```

Comment:
Maybe just use https://docs.rs/users/0.8.0/users/fn.get_user_by_uid.html until it gets added to `nix`.